ContactMechanics/FFTElasticHalfSpace.py

```diff
@@ -443,45 +588,42 @@ def evaluate_elastic_energy(self, forces, disp):
         # pylint: disable=no-self-use
         return .5 * self.pnp.dot(np.ravel(disp), np.ravel(-forces))
 
-    def evaluate_elastic_energy_k_space(self, kforces, kdisp):
+    def evaluate_scalar_product_k_space(self, ka, kb):
```

Comment:
I don't think this should be part of the FFTElasticHalfSpace class. It is more general and should go in some support module.

Reply:
Ok maybe the FFT class in  muFFT ?

Reply:
Well... it does not require any information from that object, so I wouldn't put it in the respective class. And yes, it may be useful to have this in muFFT but only as a function not a class member.